src/store/types.js

```diff
@@ -0,0 +1,2 @@
+export const GET_GAMES = 'get_games';
+export const GET_GAMESBYID = 'get_gamesbyid';
```

Comment:
- [ ] It is advantageous to define all your action types within the action creators and within one or more of the reducers.
  >  **Explanations**
      Defining the action types as constants and import them wherever used reduces the risk of accidentally misspelling them in one of the sources in which they're used, which can cause a lot of wasted time debugging a simple bug/misspelling.

- [ ] It's also important to follow the [duck pattern](https://www.freecodecamp.org/news/scaling-your-redux-app-with-ducks-6115955638be/) when defining your action types. This makes sense especially if you have concrete plans to scale up your application in the future.

Reply:
Hi @AACHOURMOHAMED, what I meant is that you should move those action types into the file you are making use of them.